TodoApp.ConsoleApp/todos.xml

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You should not commit this file into the repo.
You can create directory `data` and put the XML inside (and any other XMLs in future). Add that folder to `.gitignore`.